ui/search.html

```diff
@@ -23,79 +23,111 @@
         <div data-widget="topnavigation"><!-- --></div>
```

Comment:
You had asked me to not touch the topnavigation as you had significant changes coming through. So I just wanted to make sure that you want me to go ahead and do that.